SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,34 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember([FromBody] User user)
+        {
+            if (user.Username == null)
+            {
+                return new BadRequestObjectResult("No empty fields allowed");
+            }
+            else
+            {
+                User dbUser = _context.Users.Where(u => u.Username == user.Username)
+                    .FirstOrDefault<User>();
+
+                if (dbUser != null)
```

Comment:
Return a `NotFound()` here, if the Where returns a null it means the entry does not exist, also I agree with @ferdinandrichter, rather look the user up using the Id

Reply:
When you do this, replace the `Where` with a `Find` to use the index